test/aspnettests.js

```diff
@@ -115,10 +115,10 @@ describe('ASP.NET project file creation when kestrel command exists', function (
     before(function (done) {
         helpers.run(path.join( __dirname, '../generators/app'))
         .inTmpDir(function(dir) {
-            createTestProjectJson(dir, 'EXISTING_KESTREL_COMMAND'); })
+            createTestProjectJson(dir, 'EXISTING_WEB_COMMAND'); })
         .withLocalConfig(function() {
             return { "appInsightsOptIn": false, "runningTests": true }; })
-        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-beta8' })
+        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-rc1-final' })
```

Comment:
There is no officia RC1-update1, am I correct?
https://docs.asp.net/en/latest/search.html?q=rc1-update1&check_keywords=yes&area=default
https://github.com/aspnet/Home/releases
The rc1-update1 fixes problems, hence we are using it.